client/src/BaseForm.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Month from './Month.jsx';
+class BaseForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+
```

Comment:
If you are not planning to use a state, you can change a functional component!

Reply:
Done it!